src/chromo/models/pythia8.py

```diff
@@ -137,6 +149,34 @@ def __init__(self, evt_kin, *, seed=None, config=None, banner=True):
             f"Random:seed = {self.seed % 900_000_000}",
         ]
 
+        if evt_kin is not None:
+            # production of cache files by Pythia8 to speed up subsequent runs
+            cache_prefix = str(
```

Comment:
This should be under `if cache`, since ent_kin = None and `cache = True` would trigger an error or better `if cache and evt_kin is not None` because you need both to determine the path.